elements/map/src/plugins/advancedLayersAndSources/layers.ts

```diff
@@ -0,0 +1,11 @@
+import * as olLayers from "ol/layer";
+import STAC from "ol-stac";
+import { register } from "ol/proj/proj4.js";
+import proj4 from "proj4";
+
+register(proj4); // required to support source reprojection
```

Comment:
It seems that reprojection is only supported when the advanced plugin is imported. If this is true, we should document it in the README as well.

Reply:
Thank you, added to description!